README.md

```diff
@@ -93,3 +93,26 @@ Tag a release matching the version number `releases/swift/vX.Y.Z`
 
 1. Update the version in rust/Cargo.toml
 2. Tag a release matching the version number `releases/rust/vX.Y.Z`
+
+## Citations
+
+If you use MCAP in your research please cite it in your work. Our suggested
+citation format is:
+
+```
+@software{MCAP,
+title = {MCAP: serialization-agnostic log container file format},
+author = {{Foxglove Developers}},
+url = {https://mcap.dev},
+version = {your version},
+date = {your date of access},
+year = {2024},
+publisher = {{Foxglove Technologies}},
+address = {Online},
```

Comment:
are all of these fields needed?  `address = {Online}` doesn't seem like it is adding much value